modular_skyrat/modules/cargo/code/packs.dm

```diff
@@ -533,78 +533,3 @@
 	. = ..()
 	for(var/i in 1 to 19)
 		new /obj/item/grown/log(.)
-
-
-//////////////////////////////////////////////////////////////////////////////
-///////////////////////////// Vending Restocks ///////////////////////////////
-//////////////////////////////////////////////////////////////////////////////
-
-
-//////////////////////////////////////////////////////////////////////////////
-//////////////////////////// Temporarily Disabled ////////////////////////////
-//////////////////////////////////////////////////////////////////////////////
-
-/*
-/datum/supply_pack/vending/wardrobes/dormtime //Disabled because the dorm vendor/kinkmate is only half finished and I don't have time to fix it atm.
-	name = "Dorms-Time Restock"
-	desc = "This crate contains a refill for the Droms-Time Vendor."
-	cost = CARGO_CRATE_VALUE * 1.5
-	contains = list(/obj/item/vending_refill/kink)
-	crate_name = "dorms-time restock"
-
-/datum/supply_pack/service/wrapping_paper
```

Comment:
why was this disabled?
some people use this (me I use the wraps :))

Reply:
No idea